dev-client/src/translations/en.json

```diff
@@ -49,14 +49,16 @@
         "elevation_value": "{{value}} {{units}}",
         "elevation_unknown": "unknown",
         "not_available_offine": "Not available offline",
+        "offline": "Sites can only be created while online.",
         "empty": {
-            "info": "You don’t have any sites yet. A site lets you collect and save data, learn more about your land and its potential, and share with others.\n\n<bold>Use the map to create a site.</bold> There are three ways to use the map to find a location for a site:\n",
-            "bullet_1": "Tap the map to drop a pin and get more information about that location.",
-            "bullet_2": "Tap the location icon <icon/> to find your current GPS location.",
-            "bullet_3": "Enter coordinates, address, or other search term in the search bar."
+            "info": "<bold>Three ways to create a site:</bold>\n",
+            "map": "Tap anywhere on the map.",
+            "icon": "Tap the My Location icon.",
+            "coords": "Enter coordinates, address, or search term in the map search bar",
+            "summary": "A site lets you collect and save observations, learn more about the land and its potential, and share with others."
         },
         "create": {
-            "title": "Create Site",
+            "title": "+ Create Site",
```

Comment:
Issue: Why did a plus get added here? As far as I can tell this string is only used for the "Create site" button, which shouldn't get a plus next to it.